package.json

```diff
@@ -1,22 +1,64 @@
 {
-    "name": "skale-ima",
+    "name": "skale-ima-proxy",
```

Comment:
Why should we rename the package?

Reply:
It was the name of smart contracts package in `proxy` folder.